package.json

```diff
@@ -23,6 +23,15 @@
     "profile": "beefy tests/bench/bench.js --open -- -t brfs -t ./tests/bench/xform-rules.js -r espree",
     "coveralls": "cat ./coverage/lcov.info | coveralls"
   },
+  "gitHooks": {
+    "pre-commit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
```

Comment:
Because I'm not familiar with the "lint-staged", I'm not sure if the `eslint` command works intentionally.
This repository doesn't have `eslint` CLI command, but does this run `node bin/eslint.js` automatically?

Reply:
I was just looking at this, it seems like `lint-staged` has its own version of ESLint that it's bundled with. I need to play with it a bit to see if I can get it to use `./bin/eslint.js`.